src/components/common-components/dropdown-buttons/dropdown-buttons.component.tsx

```diff
@@ -40,40 +40,41 @@ export function DropdownButtons({
   }
 
   return (
-    <div>
+    <div className="dropdown-container">
       <Button
         variant={variant}
+        id={dataCy + "-onTopButton"}
         onClick={handleToggle}
         ref={anchorRef}
         data-cy={dataCy}
         disabled={disabled}
       >
-        {mainLabel}
-        <ArrowDropDownIcon />
+        <div className="beCenteredDamnit">
+          <div>{mainLabel}</div>
+          <div>
+            <ArrowDropDownIcon />
+          </div>
+        </div>
       </Button>
       <Popper
-        className="dropdown-buttons-z-index"
         data-cy="openedDropdown"
         open={open}
         placement="bottom"
         anchorEl={anchorRef.current}
+        disablePortal
+        id="z-index1300"
       >
-        <div
-          className={`${
-            variant === "outlined" ? "display-main-button-outlined" : "display-main-button"
-          }`}
-        >
-          <Button variant={variant}>
-            {mainLabel}
-            <ArrowDropDownIcon />
-          </Button>
-        </div>
         <ClickAwayListener onClickAway={handleClickAway}>
-          <div className="dropdown-buttons-container">
+          <div className="dropdown-buttons-list" id={dataCy}>
```

Comment:
nie wiem, czy chcemy tu używać dataCy (i to jeszcze jako id), raczej to błąd bo już jest raz użyte dataCy w tym komponencie, także też jako data-cy bym nie używał